core/src/consensus.rs

```diff
@@ -1158,35 +1159,38 @@ impl Tower {
             // Find any locked out intervals for vote accounts in this bank with
             // `lockout_interval_end` >= `last_vote`, which implies they are locked out at
             // `last_vote` on another fork.
```

Comment:
this comment looks wrong. The `lockout_interval_end` >= `last_vote` is now more obvious because the filter is transparent. This also doesn't imply they are locked out on another fork. The checks below for ancestors/root is also necessary